ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ApplicationForm/SchoolApplicationTab.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@page "/school-application-form/school-application-tab/{id:int}"
+@model ApplyToBecomeInternal.Pages.ApplicationForm.IndexModel
+@{
+    ViewData["Title"] = "School application form";
+}
+@{     
+    Layout = "~/Pages/Shared/_SchoolApplicationFormLayout.cshtml";   
+}
+
+@section BeforeMain
+{
+	<a asp-page="@Links.TaskList.Index.Page" asp-route-id="@Model.Project.Id" class="govuk-back-link">@Links.TaskList.Index.BackText</a>
+}
+
+<partial name="_SchoolApplicationHeader" model="Model.Project"/>
+
```

Comment:
Could we encapsulate more of the view in a partial view (or view component) to reduce the duplication?